llvm/tools/sycl-post-link/ModuleSplitter.cpp

```diff
@@ -336,39 +414,37 @@ namespace llvm {
 namespace module_split {
 
 std::unique_ptr<ModuleSplitterBase>
-getSplitterByKernelType(std::unique_ptr<Module> M,
-                        bool EmitOnlyKernelsAsEntryPoints,
-                        EntryPointVec *AllowedEntries) {
```

Comment:
Why is AllowedEntries removed here?

Reply:
I was more trying to understand the change here.

Reply:
Yes, exactly. In all scenarios, an entry point of a `ModuleDesc` must be a subset of the set of functions it spans, hence, AllowedEntries is actually a property of a `ModuleDesc`.